modules/game/src/test/EventTest.scala

```diff
@@ -4,6 +4,7 @@ import cats.syntax.all.*
 import chess.*
 import chess.CoreArbitraries.given
 import chess.bitboard.Bitboard
+import lila.core
```

Comment:
seems unused

Reply:
Is it not need for e.g. `lila.core.id.GameFullId` in [line 97](https://github.com/lichess-org/lila/pull/16150/files#diff-94771845f4513b49c03b971fec30f5fd9439dbccac076ec4b7fce295e569614dR97)?

Reply:
No. However you could `import lila.core.id.*` then you can just write `GameFullId`